user_manual/talk/talk_basics.rst

```diff
@@ -104,7 +134,13 @@ You can hide your video (useful during a screen share) with the little arrow jus
 
 You can access your settings and choose a different webcam, microphone and other settings in the ``...`` menu in the top bar.
 
-.. image:: images/call-menu.png
+.. image:: images/media-settings.png
+    :width: 300px
+
+.. image:: images/devices-settings.png
+    :width: 450px
+
+You can access other settings in ``Conversation settings``.
 
 .. image:: images/talk-settings.png
```

Comment:
This screenshot also could be updated, as we have a couple of new sections in this list